trace/opentelemetry/opentelemetry_test.go

```diff
@@ -343,22 +346,23 @@ func TestLogf(t *testing.T) {
 			msg:            "k1: %t, k2: %v, k3: %s",
 			fields:         map[string]interface{}{"k1": true, "k2": []bool{true}, "k3": errors.New("fake")},
 			args:           []interface{}{true, []bool{true}, errors.New("fake")},
-			expectedEvents: []sdktrace.Event{{Name: "k1: true, k2: [true], k3: fake"}},
+			expectedEvents: []trace.Event{{Name: "k1: true, k2: [true], k3: fake"}},
 			expectedAttributes: []attribute.KeyValue{
 				attribute.Bool("k1", true),
-				attribute.BoolSlice("k2", []bool{true}),
+				attribute.String("k2", fmt.Sprintf("%+v", []bool{true})),
 				attribute.String("k3", "fake"),
 			},
 		}, {
 			description:        "fatal",
 			spanName:           "test",
 			logLevel:           lFatal,
-			expectedEvents:     []sdktrace.Event{{Name: ""}},
+			expectedEvents:     []trace.Event{{Name: ""}},
 			expectedAttributes: []attribute.KeyValue{},
 		},
 	}
 
 	for _, tt := range testCases {
+		tt := tt
```

Comment:
Why do we need this?

Reply:
If tests are run in parallel the variable can be overwritten in the loop.